pkg/handlers/authorization/handler.go

```diff
@@ -79,13 +80,15 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 			}
 		}
 		req.Header.Add(headerForwardedRoles, strings.Join(context.RoleSet().List(), ","))
+		req.Header.Add(headerForwardedFor, "::1")
 	} else {
 		subject := auth.fnSubjectExtractor(req)
 		if strings.TrimSpace(subject) == "" {
 			log.Trace("Unable to determine a user's identify from certificate subject")
 			return req, errors.New("Unable to determine username")
 		}
 		req.Header.Set(headerForwardedUser, subject)
+		req.Header.Add(headerForwardedFor, "::1")
```

Comment:
Can we not add this in one place before returning the request instead of twice? Does it need to also include 127....

Reply:
updated